man/sysupdate.d.xml

```diff
@@ -869,7 +912,7 @@
       partition, and a unified kernel image to update as one. This example is an extension of the example
       discussed earlier in this man page.</para>
 
-      <para><programlisting># /usr/lib/sysupdate.d/50-verity.conf
+      <para><programlisting># /usr/lib/sysupdate.d/50-verity.transfer
 [Transfer]
```

Comment:
might have been nice to do the .conf → .transfer renaming in a separate commit. and adding .feature in a second commit.

but i understand this is quite involved splitting now, hence not asking for it.

Reply:
yeah I agree it'd be better split. I tried to split it but it wasn't trivial (i.e. just `git add -p` during a rebase) so I didn't pursue it further